en/docs/tutorials/create-your-first-scheduled-task.md

```diff
@@ -0,0 +1,221 @@
+
+The pro-code view looks as follows:
+
+!!! tip
+    To access the pro-code view, click the **<**/**>** icon.
+
+```
+import ballerina/log;
+import ballerinax/worldbank;
+
+public function main() returns error? {
+
+    worldbank:Client worldbankEndpoint = check new ({});
+    worldbank:IndicatorInformation[] getPopulationByCountryResponse = check worldbankEndpoint->getPopulationByCountry("USA");
+    anydata payload = getPopulationByCountryResponse;
+    log:printInfo((payload).toString());
+}
+```
+
+### Step 1.3: Run the scheduled task
+
+To check whether the scheduled task configuration you created is free of errors, click **Run**.
+
+![Run scheduled task](../assets/img/tutorials/scheduled-task/run-scheduled-task.png){.cInlineImage-half}
+
+If the scheduled task configuration is free of errors, the following appears in the terminal log.
+
+ ```
+ Running executable
+ ```
+
+In addition, a single log is printed with population statistics of the USA.
+
+### Step 1.4: Sync changes with Choreo Console
+
+The scheduled task configuration you created is currently only saved in the Web Editor. To deploy it and run it, you need to sync the Scheduled Task configuration to the Choreo Console by saving it in a GitHub repository. To do this, follow this procedure:
+
+!!! info
+    You are saving the scheduled task in a Choreo-maintained repository as specified in [Step 1.1: Create a new project and add a scheduled task component](#step-11-create-a-new-project-and-add-a-scheduled-task-component).
+
+1. Click **Sync with Choreo Upstream** in the bottom panel of the page (highlighted in red). In the message that appears, click **Sync my changes with Choreo**.
+
+    ![Sync with Choreo upstreeam](../assets/img/tutorials/scheduled-task/sync-with-choreo-upstream.png){.cInlineImage-full}
+
+2. In the left panel, enter a commit message (e.g., `Implement Scheduled Task`) and click on the tick.
+
+    ![Commit message](../assets/img/tutorials/scheduled-task/commit-message.png){.cInlineImage-full}
+
+    Select **Yes** in the message that appears to specify that you need the changes to be staged.
+
+3. To push the changes to the private GitHub repository maintained by Choreo, click **0↓ 1↑** in the bottom panel.
+
+    !!! tip
+        This link appears only after the system completes the committing process.
+
+    ![Push changes](../assets/img/tutorials/scheduled-task/push-changes.png){.cInlineImage-full}
+
+    Then click **OK** in the message that appears to confirm.
+
+Once the changes are successfully pushed to the GitHub repository, the Web Editor indicates by displaying the text **In sync with Choreo upstream** for the `main.bal` file as shown below.
+
+![In sync](../assets/img/tutorials/scheduled-task/in-sync-with-choreo-upstream.png){.cInlineImage-full}
+
+## Step 2: Deploy
+
+ Once you have developed a Scheduled Task and synced its configuration to the Choreo Console, you need to deploy it in order to run it. To deploy it, follow this procedure:
```

Comment:
why is scheduled Task in camel case here? Also Choreo console?
You need to deploy it to run it?